workers/unity/Packages/io.improbable.gdk.core/Subscriptions/Systems/SubscriptionSystem.cs

```diff
@@ -12,14 +11,38 @@ public class SubscriptionSystem : ComponentSystem
         private readonly Dictionary<Type, SubscriptionManagerBase> typeToSubscriptionManager =
             new Dictionary<Type, SubscriptionManagerBase>();
 
+        protected override void OnCreate()
+        {
+            base.OnCreate();
+            Enabled = false;
+
+            AutoRegisterManagers();
+        }
+
+        protected override void OnUpdate()
+        {
+        }
+
+        private void AutoRegisterManagers()
```

Comment:
Nit: Not a fan of reordering these. I do prefer the public - protected - private ordering.

I can make allowances for `OnCreate` since that's a pseudo-constructor.